### DISCUSSION TITLE
Using git to manage website files

### DISCUSSION THREAD

**Discussion by Hermansen03** [community/community#76785] [Other Feature Feedback, Questions, & Ideas]
I have a website in my ubuntu server, and remember i once worked with a guy that had the website on github. We were able to have the website on our test enviroment on our local computers and commit changes we made on the local pc using github desktop app to the server where it would automaticly pull the files when new files were commited. This would be great for me, but i've searched far and wide and cant find a way to make it do that.

I'm trying to:
1: Develop on my local machine
2: Push it using github desktop when i'm ready to update site
3: Server pulls website files to var/www

Any links, guides?

Thanks in advance

**Comment by queenofcorgis** [comment]
Hey there - I moved this to our Pages category where it will have better visibility among folks who have done this and will hopefully share their experience - thanks for posting!
